### PR TITLE
Unit test service

### PR DESCRIPTION
**Pull request by GustavoZHA**
Unit test FileStorageService
Unit test ZipService

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/core/facade/ExtractFacade.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ public static String getMetadataExtract(final MultipartFile file, final Boolean
         String pathFile = null;
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/core/facade/ExtractFacade.java:66]: The method signature should reach 120 characters

File: src/test/java/org/fundacion/jala/converter/core/ZipServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    }
+
+    @Test(expected = ZipException.class)
+    public void zipFiles_filesToZip_throwZipException() throws ZipException {
+        ArrayList<String> zipList = new ArrayList<>();
+        zipList.add(rutePath() + "archive1.txt");
+        zipList.add(rutePath() + "archive2.txt");
+        zipFiles(zipList, "\\resource34Test\\archivesFailt.zip");
+    }
+}
```
Comment (Paola-Xaq) [inline src/test/java/org/fundacion/jala/converter/core/ZipServiceTest.java]: Add an extra line at the end.

File: src/test/java/org/fundacion/jala/converter/core/FileStorageServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+ */
+public class FileStorageServiceTest {
+    FileStorageService fileStorageService = new FileStorageService();
+
+    @Test
+    public void uploadFile_MultipartFileRight_path() throws IOException, FileStorageException {
+        File file = new File("archive//archive.txt");
+        FileInputStream input = new FileInputStream(file);
+        MultipartFile multipartFile = new MockMultipartFile("archive", file.getName(), "text/archive",
+                IOUtils.toByteArray(input));
```
Comment (Paola-Xaq) [inline src/test/java/org/fundacion/jala/converter/core/FileStorageServiceTest.java:38]: Code lines can reach 120 characters, so you could move up this line

File: src/test/java/org/fundacion/jala/converter/core/FileStorageServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    @Test
+    public void getArchivePath_fileName_pathToArchive() {
+        File file = new File("archive/testPath2");
+        String actual = fileStorageService.getArchivePath("testPath2");
+        String expected = file.getAbsolutePath();
+        assertEquals(expected, actual);
+    }
+}
```
Comment (Paola-Xaq) [inline src/test/java/org/fundacion/jala/converter/core/FileStorageServiceTest.java:88]: Add an extra line at the end.

File: src/test/resource/archive2.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+Hello AT13
```
Comment (cristianfchq) [inline src/test/resource/archive2.txt:1]: I think that this  file is not necesary

File: src/test/resource/archive1.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+Hello world
```
Comment (cristianfchq) [inline src/test/resource/archive1.txt:1]: I think that this file is not necesary

File: src/test/java/org/fundacion/jala/converter/core/FileStorageServiceTest.java (hunk 3 above)
Comment (GustavoZHA) [reply src/test/java/org/fundacion/jala/converter/core/FileStorageServiceTest.java:38]: it is not possible, if I could move up that line, it past 120 character

File: src/test/java/org/fundacion/jala/converter/core/ZipServiceTest.java (hunk 2 above)
Comment (GustavoZHA) [reply src/test/java/org/fundacion/jala/converter/core/ZipServiceTest.java]: added

File: src/main/java/org/fundacion/jala/converter/core/facade/ExtractFacade.java (hunk 1 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/core/facade/ExtractFacade.java:66]: Solved

File: src/test/resource/archive2.txt (hunk 5 above)
Comment (GustavoZHA) [reply src/test/resource/archive2.txt:1]: To zip a file this file have to be empty

File: src/test/resource/archive1.txt (hunk 6 above)
Comment (GustavoZHA) [reply src/test/resource/archive1.txt:1]: To zip a file this file have to be empty

**Review by danielaSantaCruz — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Review by Paola-Xaq — APPROVED** [review]

**Pull request merged**